iOSIDPTemplate/Images.xcassets/sam.imageset/Contents.json

```diff
@@ -0,0 +1,21 @@
+{
+  "images" : [
+    {
+      "idiom" : "universal",
+      "filename" : "User2_2.png",
```

Comment:
Why do we need those user pics for? Shouldn't we show the user's actual profile picture?

Reply:
@trooper2013 I thought those pics were just for the demo?

Reply:
The actual user's icon requires an extra API call against `FileForce`, so we decided to use static images IIRC.